packages/client/src/utils/utils.ts

```diff
@@ -139,8 +154,7 @@ export function convertPeerDescriptorToNetworkPeerDescriptor(descriptor: PeerDes
         throw new Error('nodeType "virtual" not supported')
     }
     return {
-        ...omit(descriptor, 'kademliaId'),
-        id: binaryToHex(descriptor.kademliaId),
+        nodeId: binaryToHex(descriptor.nodeId),
         type: descriptor.type === NodeType.NODEJS ? NetworkNodeType.NODEJS : NetworkNodeType.BROWSER
```

Comment:
Doesn't necessarily include `websocket` field (or others)?

(Missing unit test for this?)